README.md

```diff
@@ -61,16 +61,23 @@ cd codefund/
 # Optionally set the following environment variables (see .env-sample)
 # config postgres in config/dev.exs
 # start postgres
+gem install fakes3
+
 mix deps.get
 mix ecto.create
 mix ecto.migrate
 mix code_fund.seed
 cd assets && npm install
 cd ../
 mix maxmind.setup
+fakes3 -r /tmp/fakes3_root -p 4567
 mix phx.server
 ```
 
+#### Running Tests
+We use `fakes3` for mocking S3 uploads so you will need to have this service running prior to running tests.
+The way to do this is by running `fakes3 -r /tmp/fakes3_root -p 4567`
```

Comment:
Do you think we should add a script (bin/test) that will perform both this and the test call?